### DISCUSSION TITLE
Application icon

### DISCUSSION THREAD

**Discussion by vladrevers** [FossifyOrg/General-Discussion#611] [Ideas]
The icon for the notes app as a book looks weird, please look at the "note" and "sticker" icons

For example these:
![icons](https://github.com/FossifyOrg/Notes/assets/56305121/984c342b-ef1f-4229-8350-f6115f3e15d6)

Sources (downloaded in svg/xml):
- https://pictogrammers.com/library/mdi/icon/sticker-text-outline/
- https://pictogrammers.com/library/mdi/icon/sticker-text/
- https://pictogrammers.com/library/mdi/icon/note-text-outline/
- https://pictogrammers.com/library/mdi/icon/note-text/

**Comment by tswistak** [comment]
I agree. IMHO the current icon would suit more an e-book reader or maybe even the [PDF viewer](https://github.com/FossifyOrg/File-Manager/discussions/12).

**Comment by naveensingh** [comment]
Yes, it'll be updated but the suggested document icon will be used for the PDF viewer.

**Comment by naveensingh** [comment]
https://github.com/FossifyOrg/artwork/blob/main/Icons/notes.png
